AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -1080,6 +1087,14 @@ - (ASLayout *)calculateLayoutThatFits:(ASSizeRange)constrainedSize
 
   ASDN::MutexLocker l(__instanceLock__);
 
+#if YOGA /* YOGA */
+  if (ASHierarchyStateIncludesYogaLayoutEnabled(_hierarchyState) == YES &&
```

Comment:
This integration in the layout method is just an early-bail condition that has no other ties into the behavior of the method.  So, total refactoring of the rest of the method should be OK.

If any issues are encountered when refactoring, I am very supportive of deleting this integration entirely, and just ask me to figure out how to re-add it with the new code.  The intention is not to add more officially-supported paths.